### PR TITLE
Bugfix in urls and improvemenets search

### PR DESCRIPTION
**Pull request by szok**
- Repair bugfix with copying URLs
- Add search query value to URL

### CODE REVIEW

File: ui/static/app/auth/auth.service.ts [hunk 1]
```diff
@@ -61,7 +61,7 @@ export class AuthService {
   }
 
   isAdmin(): boolean {
-    return (this.localStorage.get("is_admin") === "true") ? true : false;
+    return (this.localStorage.get("is_admin") === "true");
```
Comment (mkurek) [inline ui/static/app/auth/auth.service.ts:64]: 👍

File: ui/static/app/record/record.component.ts [hunk 2]
```diff
@@ -64,21 +64,27 @@ export class RecordComponent extends SearchComponent implements OnInit {
     this.additionalRouteParams["showAll"] = this.routeParams.get("showAll");
     let url_offset: string = this.routeParams.get("offset");
     this.currentOffset = url_offset ? Number(url_offset) : 0;
-    this.searchValue = this.routeParams.get("search");
+    let search: string = this.routeParams.get("search");
+    this.searchValue = (search !== "null") ? search : "";
```
Comment (mkurek) [inline ui/static/app/record/record.component.ts]: when is the case when null will be passed as a string? cannot we pass empty string as a query string then?

File: ui/static/app/record/record.component.ts [hunk 3]
```diff
@@ -64,21 +64,27 @@ export class RecordComponent extends SearchComponent implements OnInit {
...
     this.getRecords();
   }
 
   search(value: string) {
     this.currentOffset = 0;
     if (value.length > 1) {
       this.searchValue = value;
-      this.getRecords();
+      this.searchUpdateUrls();
```
Comment (mkurek) [inline ui/static/app/record/record.component.ts]: this could be done after `if`

**Comment by mkurek** [comment]
:ok:

**Comment by ghost** [comment]
:ok:

**Pull request merged**